### ISSUE TITLE
Closing the app on Mac

### ISSUE THREAD

**Issue by klauslovgreen** [lightninglabs/lightning-app#205]
When closing the app (x) it continues to run in the background which might be intended? However, when clicking the Lightning icon in the dock the app does not re-appear and you need to kill the app and start again (and re-sync)

**Comment by Roasbeef** [comment]
> When closing the app (x) it continues to run in the background which might be intended? 

Clicking "x" on the app _should_ cause it to exit completely. In the future the app will likely fold into an icon on the main toolbar.

**Comment by klauslovgreen** [comment]
Just tried 0.15 alpha - same experience exactly. I close the app clicking "x"  - the Lightning app icon remains in the dock and both Lightning and lnd keeps on running in the background. 

When I click the Lightning App icon in the dock - the app window does not re-appear and the only way is to right-click the icon is the dock and click "Quit".

I also noticed that the very first time I run the app (this time on a new Mac) - the app does not actually open the app window at all - just the icon is in the Dock but noting happens. I need to quit the app and then re-start it fo the app to appear and start syncing.

I have deleted the "lnd" directory when using a new version.
